### ISSUE TITLE
import/no-internal-modules is warn/error for relative paths

### ISSUE THREAD

**Issue by stefan-schweiger** [iamturns/eslint-config-airbnb-typescript#205]
As soon as I use this package with an active `import/no-internal-modules` configuration I'm getting warn/error messages for relative path imports.

```
"import/no-internal-modules": [
  "error",
  {
    "allow": [
      "@angular/*",
      "@angular/cdk/*",
      "@angular/material/*",
      "@angular/platform-browser/*",
      "@angular/**/testing",
      "@backend/*",
      "@frontend/*"
    ]
  }
],
```

The config above without `eslint-config-airbnb-typescript` only shows error when I'm importing a not specified submodule.

```
import { Component } from @angular/core; // works
import * from @angular/localize/init; // error as expected
import { Test } from './test-helper' // works
import { TestParent } from '../parent-test-helper' // works
```

But as soon as I use `eslint-config-airbnb-typescript` this is the result:

```
import { Component } from @angular/core; // works
import * from @angular/localize/init; // error as expected
import { Test } from './test-helper' // error but should work
import { TestParent } from '../parent-test-helper' // error but should work
```

I've also tried to  add patterns to mitigate this but this also did nothing for me:
```
"allow": [
   // ...
  "./*",
  "./**/*",
  "../*",
  "../**/*",
]
```
